app/services/flowcommerce_spree/import_item.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+module FlowcommerceSpree
```

Comment:
Shouldn't there be specs for this class?

Reply:
You are right. I will try adding tests for this right away!

Reply:
Added some tests for this model!